src/manifolds/VectorBundle.jl

```diff
@@ -169,6 +201,55 @@ struct VectorBundleBasisData{BBasis<:CachedBasis,TBasis<:CachedBasis}
     vec_basis::TBasis
 end
 
+@doc raw"""
+    struct VectorBundleInverseProductRetraction <: AbstractInverseRetractionMethod end
+
+Inverse retraction of the point `y` at point `p` from
+vector bundle `B` over manifold `B.fiber` (denoted $\mathcal M$).
+
+Notation:
+  * The point $p = (x_p, V_p)$ where $x_p ∈ \mathcal M$ and $V_p$ belongs to the
+    fiber $F=π^{-1}(\{x_p\})$ of the vector bundle $B$ where $π$ is the
+    canonical projection of that vector bundle $B$.
+    Similarly, $q = (x_q, V_q)$.
+
+The inverse retraction is calculated as
+
+$\log_p q = (\log_{x_p}(x_q), V_{\log} - V_p)$
+
+where $V_{\log}$ is the result of vector transport of $V_q$ to the point $x_p$.
+The difference $V_{\log} - V_p$ corresponds to the logarithmic map in the vector space $F$.
+
+See also [`VectorBundleProductRetraction`](@ref).
+"""
+struct VectorBundleInverseProductRetraction <: AbstractInverseRetractionMethod end
+
+@doc raw"""
+    struct VectorBundleProductRetraction <: AbstractRetractionMethod end
```

Comment:
Same as above, currently an inexperienced user (or a reviewer with a little bit of jet lag) might wonder why `Product` is in the name.